analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UseDateTimeInsteadOfDateTimeOffset.CSharp9.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Globalization;
+
+DateTime myDate = new(1); // Noncompliant {{Prefer using "DateTimeOffset" struct instead of "DateTime"}}
+//                ^^^^^^
+_ = new DateTime(1, 1, 1, 1, 1, 1, 1, 1, new GregorianCalendar());                   // Noncompliant
```

Comment:
These constructors are available in the .NET Framework and they don't use any kind of expression that isn't available in earlier C# versions. Why is it placed in this test file?

Reply:
If run those 3 constructors in .NET Framework they results in:
`build error [CS1503]: Argument 8: cannot convert from 'int' to 'System.Globalization.Calendar'`
or
`build error [CS1729]: 'DateTime' does not contain a constructor that takes 10 arguments`